internal/config/config.go

```diff
@@ -207,6 +220,32 @@ func Initialize(configFiles ...string) {
 			config.Unleash.Token = *cfg.FeatureFlags.ClientAccessToken
 		}
 
+		// kafka
+		config.Kafka.Brokers = make([]string, len(cfg.Kafka.Brokers))
+		for i, b := range cfg.Kafka.Brokers {
+			config.Kafka.Brokers[i] = fmt.Sprintf("%s:%d", b.Hostname, b.Port)
+
+			// assumption: TLS/SASL credentials are always the same for all nodes in a cluster
+			if b.Authtype != nil && *b.Authtype != "" {
+				config.Kafka.AuthType = string(*b.Authtype)
+			}
+			if b.Cacert != nil && *b.Cacert != "" {
+				config.Kafka.CACert = *b.Cacert
+			}
+			if b.Sasl.SecurityProtocol != nil && *b.Sasl.SecurityProtocol != "" {
+				config.Kafka.SASL.SecurityProtocol = *b.Sasl.SecurityProtocol
+			}
+			if b.Sasl.SaslMechanism != nil && *b.Sasl.SaslMechanism != "" {
+				config.Kafka.SASL.SaslMechanism = *b.Sasl.SaslMechanism
+			}
+			if b.Sasl.Username != nil && *b.Sasl.Username != "" {
+				config.Kafka.SASL.Username = *b.Sasl.Username
+			}
+			if b.Sasl.Password != nil && *b.Sasl.Password != "" {
+				config.Kafka.SASL.Password = *b.Sasl.Password
+			}
+		}
+
```

Comment:
Clowder passes in multiple brokers, each one of them with a unique TLS CA and SASL credentials. I am assuming that they are all the same, all other services I looked up simply use the first server and ignore the rest. That is weird, nobody was able to answer me questions in this regard. So let's try this in stage to see what happens. Likely this can fail, then we would also use only the first server.